toketmaster/src/client.py

```diff
@@ -0,0 +1,19 @@
+from algosdk.v2client import algod, indexer
+
+
+_algod_token = "LGlsGlaSiY37Y3xKQly071AsDwS6PeP73mAaFJLF"
+_headers = {
+        "X-API-Key": _algod_token,
+    }
+
+
+algod_client = algod.AlgodClient(
+    algod_token=_algod_token, 
+    algod_address="https://testnet-algorand.api.purestake.io/ps2",
+    headers=_headers)
+
+
+indexer_client = indexer.IndexerClient(
```

Comment:
Nice, lets move this to https://github.com/algocertified/sandbox/blob/master/toketmaster/fixtures.py.

Reply:
https://github.com/algocertified/sandbox/pull/2#discussion_r655894674
I am fine moving it to a different module, but I don't want the clients to be fixtures